install.sh

```diff
@@ -258,6 +258,7 @@ set_synchrony_url() {
     helm upgrade confluence atlassian-data-center/confluence -n atlassian --reuse-values --set synchrony.ingressUrl="${SYNCHRONY_FULL_URL}" > /dev/null
     log "Synchrony URL is set to '${SYNCHRONY_FULL_URL}'."
   fi
+  helm list --namespace atlassian
```

Comment:
Can we move it out of synchrony function?

Reply:
Yes, moved it at the end of script.